src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -144,16 +126,29 @@ export default function applyAuthInterface(httpClient, authConfig) {
         // Attempt to refresh the JWT cookies.
         httpClient
           .refreshAccessToken()
-          // Successfully refreshed the JWT cookies, fire the callback function.
+          // Successfully refreshed the JWT cookies
           .then((response) => {
             const refreshedAccessToken = httpClient.getDecodedAccessToken();
 
-            /* istanbul ignore next */
             if (refreshedAccessToken === null) {
-              logMissingAccessToken(httpClient, accessToken, response);
+              // TODO: Why does the api call sometimes succeed, but the JWT cookie isn't found?
+              const error = new Error('Access token is null after refresh.');
```

Comment:
Question: Logging out the user seems like a heavy workaround and a far from ideal user experience.  Have we done further logging on the LMS side to understand why the `refreshAccessToken` is not returning a JWT?

Reply:
I agree that we need to dig more and learn what the problem is, but by returning null today I think we're creating errors in the application.

Reply:
Agree that I still have some digging to try to rid ourselves of this altogether.

In the meantime, if we do get into this situation, we can either:
1. Logout and try to the clean-up auth, or
2. Raise an error.
I see pros/cons with both and chose one.  @nasthagiri. let me know if you want this to switch to raising an error.

Reply:
What is the user experience of these 2 paths?

I know from past experience that abruptly taking the user to the login page isn't that great.

Reply:
Done. We decided to throw an error.